src/translations/en.global.json

```diff
@@ -66,6 +66,9 @@
         },
         "deleteRecoveryMethod": {
             "modal": "You will need to confirm delete key details on your Ledger."
+        },
+        "LedgerPublicKey": {
```

Comment:
@Patrick1904 where is this string used. I wonder if I'm confused or some file is missing from commit.

Reply:
@vgrichina this what [`modal.textId`](https://github.com/near/near-wallet/blob/ledger-create-account/src/components/accounts/ledger/LedgerConfirmActionModal.js#L24) returns for the `Translate` `id` in the `LedgerConfirmActionModal` while requesting the Ledger key